src/app/src/components/OARMap.jsx

```diff
@@ -190,7 +190,10 @@ class OARMap extends Component {
                 0,
                 (err, geojson) => {
                     if (!err) {
-                        this.oarMap.fitBounds(getBBoxForArrayOfGeoJSONPoints(geojson));
+                        this.oarMap.fitBounds(
+                            getBBoxForArrayOfGeoJSONPoints(geojson),
+                            { padding: 50 },
```

Comment:
It would be helpful to add a commit message explaining why we need this padding and how we chose 50.

Reply:
Sounds good -- will update. I added this because I noticed in the demo gif that when it zoomed to the bbox, points on the edge of the bbox would be on the edge of the viewport, or nearly out of view.

Reply:
Haven't updated this yet but I will make sure to do so when rebasing before merging.